src/pages/profile/Profile.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useAuthService } from 'hooks';
+
+const Profile = () => {
+  const { logout } = useAuthService();
+  return (
+    <div className="profile">
+      <h1>Name: Manish</h1>
+      <h1>Email: manishdevrani777@gmail.com</h1>
```

Comment:
try to fetch user email from firebase rather than hard coding it

Reply:
Ok, thanks for pointing